providers/certification-client/units/client-cert-odm-desktop-20-04.pxu

```diff
@@ -61,6 +61,7 @@ nested_part:
     after-suspend-usb-c-cert-full # no manual only
     # after-suspend-wireless-cert-full # auto only
     info-attachment-cert-full
+    acpi_oem_osi_strings
```

Comment:
If you implement a `acpi-manual` test plan, you can put it here instead of `acpi_oem_osi_strings` (
it's better to put it higher in the list, maybe above `audio-manual`):

```suggestion
    acpi-manual
```


If you don't, then you can just remove `acpi_oem_osi_strings`.